CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ For semantic convention package changes, see the [semconv CHANGELOG](packages/se
 ### :bug: (Bug Fix)
 
 * fix(sdk-trace-base): do not load OTEL_ env vars on module load, but when needed [#5224](https://github.com/open-telemetry/opentelemetry-js/pull/5224)
+* fix(instrumentation-xhr, instrumentation-fetch): content length attributes no longer get removed with `ignoreNetworkEvents: true` being set [#5229](https://github.com/open-telemetry/opentelemetry-js/issues/5229)
```

Comment:
sorry for our confusing changelog setup :/
this should go into `CHANGELOG_NEXT.md` on this branch. 

The fix from `main` will already be included once this version is released (February), so you can just document the breaking change to `sdk-trace-web` as `feat(sdk-trace-web)!: ...` 🙂

Reply:
Let me know if it looks good now. I had a bunch of meetings but am free now.